web/src/popover_menus.ts

```diff
@@ -302,7 +302,7 @@ function get_props_for_popover_centering(
     return {
         arrow: false,
         getReferenceClientRect: () => new DOMRect(0, 0, 0, 0),
-        placement: "top",
+        placement: "bottom",
```

Comment:
This should be skipped, or a separate change with a separate commit message justifying it, since it's likely to negatively impact the user experience for some popovers where the designer intent is for to be placed above the element?

Reply:
After more debugging, it seems like placement: "bottom" is what actually fixes the issue. I still could not figure out where the core of the issue lies, but I tried one possible fix that just makes sure placement: "bottom" is passed as an overriding prop, without changing the default behavior of get_props_for_popover_centering. Because when you click the reaction button, it's already passing placement: "bottom" as a prop, I'm just making sure that reaches the final input to the tippy object. Not sure if that's still the best way to do it, or if it can cause issues for other popovers

With that being said I think I completely messed up and added all of the commits that I fetched into my branch into this pull request.... sorry about that

Reply:
Would it be best for me to open a new pull request instead of trying to fix this one?

Reply:
No, it's pretty much always best to push changes to the same PR, to preserve comment history.